scripts/deploy_assisted_service.sh

```diff
@@ -14,6 +14,7 @@ export PROFILE=${PROFILE:-assisted-installer}
 export OCP_SERVICE_PORT=$(( 7000 + $NAMESPACE_INDEX ))
 export OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE:-}
 export PUBLIC_CONTAINER_REGISTRIES=${PUBLIC_CONTAINER_REGISTRIES:-}
+export ENABLE_KUBE_API=${ENABLE_KUBE_API:-}
```

Comment:
i would expect it to be a boolean and not a full command that user need to set.

Reply:
sounds good